posts/jhipster-5-release.md

```diff
@@ -0,0 +1,196 @@
+
+You can see [the GitHub](https://github.com/jhipster/generator-jhipster/issues/6044) for more details.
+
+
+## Angular support
+
+The whole support for Angular has been improved in JHipster 5 with multiple optimizations and rework. The main changes are the focus on Angular 6 and the deletion of popup dialogs for the entity screens.
+
+Here is a detailed list of all the changes related to Angular:
+- JHipster is focusing on Angular 6, that means the support for the older AngularJS 1.x has been removed
+- As a side effect, the support for Gulp and Bower has been removed as well
+- Migration to Webpack 4 (both Angular and React), which gives a nice performance boost
+- The admin modules are now lazy loaded, that speeds up the initial load of the application
+- The folder structure of entities has been improved, especially for microservices as now they are grouped by services. You can use a new flag `--skip-ui-grouping` to retain the old behavior
+- The entity screen is not using popups anymore for creation/edition
+
+[![jhipster5-entity-screen](https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2018/06/jhipster5-entity-screen.png)](https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2018/06/jhipster5-entity-screen.png)
+
+
+## Misc:
+
+Here the miscellaneous changes:
+- Both Angular and React now uses Prettier to format code
+- Migration from PhantomJS to Jest as PhantomJS is not developed anymore. The frontend tests are now using the Headless Chromium browser provided by Puppeteer
+
+
+# Backend changes
+
+## Spring Boot 2
+
+On the backend side, the biggest change is the upgrade of Spring Boot to the latest major version. Spring Boot 2 was released on March 1st and upgrades all the major Spring dependencies, the main one being the newest Spring Framework 5.
+
+The JHipster team has put a lot of effort to upgrade the generated java classes to Spring Boot and the updated dependencies.
+
+
+Among the new cool features, we can list:
+
+- Java 8 is the baseline and Java 9 is supported
+- Spring web MVC is now an optional dependency
+- Gradle plugin has been rewritten
+- HTTP/2 for Tomcat, Undertow and Jetty
+- Security is auto-configured by Spring-Boot auto-configure (unless JHipster overrides it)
+- new Actuators endpoints
+
+
+### Spring Cloud
+
+With the upgrade to Spring boot 2, JHipster also brings the latest Spring Cloud release train support: Finchley
+
+The Spring Cloud projects are the cornerstone of the JHipster microservices architecture. The Finchley release upgrades all the Spring Cloud projects to the latest and introduces new components. The JHipster team has migrated all the dependencies following this new release train and has fixed the regressions for you.
+
+### Kotlin support
+
+This is still in early stage, but the team always follows the latest trends and has announced a Kotlin support for the backend!
```

Comment:
It would be awesome to explain what blueprints are and the possibility they offer as they have been redesigned in JHipster 5. See: https://www.jhipster.tech/modules/creating-a-blueprint/